src/docs/forms/components/TextInput/examples/02-SimpleTextInput.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
```

Comment:
We cannot use this example since it uses the `onKeyUp` property which is internal. Please change the example so that it disables the component when the value becomes a certain 'sentinel' value instead.